app/models/validations/sales/soft_validations.rb

```diff
@@ -123,6 +123,19 @@ def discounted_ownership_value_invalid?
     mortgage_deposit_and_grant_total != value_with_discount && discounted_ownership_sale?
   end
 
+  def buyer1_not_livein?
+    return unless ownershipsch && buy1livein
+
+    (discounted_ownership_sale? || shared_ownership_scheme?) && buy1livein == 2
+  end
+
+  def buyer2_not_livein?
+    return unless ownershipsch && buy2livein
+    return unless joint_purchase?
+
+    (discounted_ownership_sale? || shared_ownership_scheme?) && buy2livein == 2
+  end
+
```

Comment:
https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1512/files#diff-896f324e0f3c60b69ebafa248b8305404cea7cb90b44c1632c2f007b9973160f

this hasn't been PO'ed and merged yet, but if it is in time there will be methods to reuse here

Reply:
Yes!